ETC.tsv

```diff
@@ -8114,9 +8114,9 @@ ETC_20150317_008114	Kepa AI
 ETC_20150317_008115	Kepa AI2
 ETC_20150317_008116	Strike
 ETC_20150317_008117	Chapparition's Shadow
-ETC_20150317_008118	Vubbe Fighter is very exhausted.
-ETC_20150317_008119	Destroy the summoning post that Specter Monarch is casting
-ETC_20150317_008120	If it is not destroyed, the Monarch's lighting will stay continuous.
+ETC_20150317_008118	The Vubbe Fighter is very exhausted.
+ETC_20150317_008119	Destroy the summons the Specter Monarch summoned
+ETC_20150317_008120	If not, the Monarch's lightning will never end.
```

Comment:
If not, the lightning will never end.

or

If not, the lightning from the Specter Monach will never end.